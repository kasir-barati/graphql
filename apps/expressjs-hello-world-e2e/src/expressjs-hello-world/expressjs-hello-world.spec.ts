import axios from 'axios';

describe('GET /', () => {
  it('should return "Hello world"', async () => {
    const query = `{ hello }`;

    // http://localhost:4000/graphql
    const res = await axios.post(
      '/graphql',
      { query },
      {
        headers: {
          'content-type': 'application/json',
        },
      },
    );

    expect(res.status).toBe(200);
    expect(res.data).toMatchSnapshot();
  });
});
