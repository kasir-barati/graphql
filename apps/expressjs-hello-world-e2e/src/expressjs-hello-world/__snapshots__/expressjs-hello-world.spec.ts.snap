// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET / should return "Hello world" 1`] = `
{
  "data": {
    "hello": "Hello world",
  },
}
`;
